hamster_gtk/overview/dialogs/overview_dialog.py

```diff
@@ -138,12 +138,15 @@ def _on_charts_button(self, button):
             self._charts.destroy()
             self._charts = False
         else:
-            self._charts = widgets.Charts(self._totals)
+            self._charts = Gtk.ScrolledWindow()
+            self._charts.set_min_content_height(200)
```

Comment:
Otherwise the scrolled Window will not expand has the content technically can be displayed in a very small window due to scolling.